docs/notes/2.23.x.md

```diff
@@ -12,6 +12,12 @@ We offer [formal sponsorship tiers for companies](https://www.pantsbuild.org/spo
 
 ### Highlights
 
+- New backends for nFPM, Trufflehog and buildifier
```

Comment:
Maybe note they are experimental? Or a call to action to take them for a spin?

Reply:
Good ideas, done both.